pandas/tests/test_series.py

```diff
@@ -5021,7 +5021,10 @@ def test_getitem_setitem_datetime_tz_pytz(self):
     def test_getitem_setitem_datetime_tz_dateutil(self):
         tm._skip_if_no_dateutil();
         from dateutil.tz import tzutc
-        from dateutil.zoneinfo import gettz
+        if sys.platform != 'win32':
```

Comment:
maybe make a function in ``pandas/compat/__init__.py`` like: ``is_platform_windows()/mac/linux`` just for convenience